src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7897,6 +7964,7 @@ public partial class UnobservedTaskExceptionEventArgs : System.EventArgs
     [System.Runtime.CompilerServices.AsyncMethodBuilderAttribute(typeof(System.Runtime.CompilerServices.AsyncValueTaskMethodBuilder<>))]
     public readonly partial struct ValueTask<TResult> : System.IEquatable<System.Threading.Tasks.ValueTask<TResult>>
     {
+        internal readonly TResult _result;
```

Comment:
Should all the fields be private?

Reply:
I noticed a couple of these as well, I don't know why they showed up as internal but I will investigate.

Reply:
The reason this and a couple others are internal instead of private is because for any non-public fields that are generic I just blindly include those unchanged. For now I will keep them like this so the correctly show the usage of generic parameters. If you feel there is a reason to switch them I can.

Reply:
ok

Reply:
Same in `Nullable<T>`    -   `internal T value` .   I think that is ok. 

---
In reply to: [161028771](https://github.com/dotnet/corefx/pull/26264#discussion_r161028771) [](ancestors = 161028771)